requirements/lint.txt

```diff
@@ -1,7 +1,7 @@
 -r pip.txt
 maxcdn
-astroid<1.4
-pylint<1.5
+astroid
+pylint
 prospector
-pylint-django<0.7
-pyflakes<1.2.0
```

Comment:
I haven't run into any issues other than more linting errors. We were only pinned because we hit a bug with the latest version at some point. We should be pinned to something more recent.